__tests__/app.test.js

```diff
@@ -84,6 +84,45 @@ describe("app", () => {
         });
     });
   });
+
+  describe("PATCH - /api/articles/:article_id", () => {
+    test("status: 202 - should accept the request in the form of an object { inc_vote: newVote }", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 10 })
+        .expect(201)
+        .then((res) => {
+          expect(res.body.article.votes).toBe(110);
+        });
+    });
```

Comment:
You can have another happy path test to ensure negative values are still accepted